src/web/lib/hakuneko/frontend@modern/app.css

```diff
@@ -0,0 +1,104 @@
+:host {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+.app {
+  display: flex;
+  flex-direction: row;
+  height: 100%;
+  width: 100%;
+  overflow: hidden;
+  background: var(--base);
```

Comment:
Variable name is not expressive, someone who is going to change this in the corresponding file probably have no idea where this is used.
Variables should better describe their usage, e.g. `--main-panel-background-color`